assets/stylesheets/_components.scss

```diff
@@ -34,6 +34,7 @@
 @import 'components/button/style';
 @import 'components/card/style';
 @import 'components/credit-card-form-fields/style';
+@import 'components/credit-card-form-fields-ux/style';
```

Comment:
We are moving `stylesheets` from this file. Please check @jsnajdr work [here](https://github.com/Automattic/wp-calypso/pulls?q=is%3Apr+is%3Aclosed+author%3Ajsnajdr+label%3A%22CSS+Migration%22).

Please update the code when the test finishes.

Reply:
Thanks for the heads up! Will do :)